### PR TITLE
This commit covers a couple changes on eap 6 and 7 integration tests, for more details check the commit message

### PR DESCRIPTION
**Pull request by spolti**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,8 +28,8 @@
         <version.arquillian>1.1.11.Final</version.arquillian>
         <version.org.wildfly.arquillian>1.0.2.Final</version.org.wildfly.arquillian>
         <version.arquillian.governor>1.0.1.Final</version.arquillian.governor>
-        <version.eap6>7.5.11.Final-redhat-SNAPSHOT</version.eap6>
-        <version.eap7>7.0.0.GA-redhat-2</version.eap7> <!-- eap-7.0.0 -->
+        <version.eap6>7.5.11.Final-redhat-SNAPSHOT</version.eap6> <!--eap 6.4.x downstream -->
+        <version.eap7>7.0.3.GA-redhat-SNAPSHOT</version.eap7> <!-- eap-7 downstream-->
         <org.jboss.bom.version>7.0.0.GA</org.jboss.bom.version>
```
Comment (jwendell) [inline pom.xml]: General newbie question: is it really good to depend on SNAPSHOT versions? Can't we depend on released ones?

File: pom.xml [hunk 2]
```diff
@@ -116,7 +116,7 @@
                 <updatePolicy>always</updatePolicy>
             </snapshots>
         </repository>
-        <!-- <repository> <id>1rhat</id> <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</url> 
+        <!-- <repository> <id>1rhat</id> <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</url>
             </repository> -->
         <repository>
```
Comment (jwendell) [inline pom.xml]: Can't we just remove those comments?

File: pom.xml [hunk 3]
```diff
@@ -127,7 +127,7 @@
             <id>3rhat</id>
             <url>https://maven.repository.redhat.com/earlyaccess/all/</url>
         </repository>
-        <!-- <repository> <id>4rhat</id> <url>https://repository.jboss.org/</url> 
+        <!-- <repository> <id>4rhat</id> <url>https://repository.jboss.org/</url>
             </repository> -->
         <repository>
```
Comment (jwendell) [inline pom.xml]: Ditto

File: pom.xml [hunk 4]
```diff
@@ -141,7 +141,7 @@
             <id>7rhat</id>
             <url>https://repository.jboss.org/nexus/content/repositories/thirdparty-uploads/</url>
         </repository>
-        <!-- <repository> <id>8rhat</id> <url>http://repository.jboss.org/nexus/content/groups/developer/</url> 
+        <!-- <repository> <id>8rhat</id> <url>http://repository.jboss.org/nexus/content/groups/developer/</url>
             </repository> -->
     </repositories>
```
Comment (jwendell) [inline pom.xml]: Ditto.

**Comment by rcernich** [comment]
We need to stop using commit messages like, "little fixes."  The comment should describe, briefly, what is being changed, e.g. "update dependency versions for eap 6 and 7."

File: pom.xml (hunk 1 above)
Comment (spolti) [inline pom.xml]: I don't think it is a good idea until the changes needed to the integration tests get working is not merged, For now I think we can just do like Kevin suggested, fork in within our repo and use it.

**Pull request merged**